.github/workflows/ci.yml

```
@@ -22,10 +22,10 @@ jobs:
           - version: '1'  # The latest point-release (Windows)
             os: windows-latest
             arch: x64
-          - version: '1.9'  # 1.9 
+          - version: '1.10'  # 1.10 
```

I think this should work, resolving to the current LTS (now 1.10)
```suggestion
          - version: 'lts' 
```